web/thesauruses/haskell/control_structures.json

```diff
@@ -6,92 +6,53 @@
   },
   "categories": {
     "Conditional Statements (ifs)": [
-      "if_conditional",
       "if_else_conditional",
-      "if_elseif_conditional",
       "if_elseif_else_conditional",
+      "switch_statement",
       "ternary_conditional"
     ],
     "Loops": [
       "while_loop",
-      "do_while_loop",
       "until_loop",
-      "do_until_loop",
-      "for_loop",
       "foreach_loop"
     ],
     "Iterations": [
-      "each_iteration",
       "map_iteration",
       "filter_iteration",
-      "fold_iteration",
-      "recursion_iteration"
+      "fold_iteration"
     ]
   },
   "control_structures": {
-    "if_conditional": {
-      "name": "If conditional",
-      "code": ""
-    },
```

Comment:
I'm removing all those empty code blocks, they result in "empty blocks" on the reference sheets.